src/reference/modules/installation/pages/upgrade.adoc

```diff
@@ -18,6 +18,14 @@ the plugin.
 Read https://rudder.io/plugins[the plugins page on our website] for more information.
 ====
 
+[WARNING]
+====
+When upgrading from a 4.1 older than 4.1.13, a 4.2 older than 4.2.7 or a 4.3 older than 4.3.3,
+it is necessary to explicitely upgrade `rudder-agent` (in the same command as 
+the other packages) when upgrading a relay or a root server,
+because of a dependency misconfiguration in the agent package.
```

Comment:
You don't specify on which system and why we cannot solve solve it (the cause is in the old package)